pkg/kn/commands/human_readable_flags.go

```diff
@@ -113,3 +114,8 @@ func TranslateTimestampSince(timestamp metav1.Time) string {
 	}
 	return duration.HumanDuration(time.Since(timestamp.Time))
 }
+
+// AddGitOpsFlags adds flags to enable gitops mode
+func AddGitOpsFlags(flags *pflag.FlagSet) {
+	flags.String("target", "", "work on local directory instead of a remote cluster")
```

Comment:
A little formatting nit to align with the rest.
```suggestion
	flags.String("target", "", "Work on local directory instead of a remote cluster.")
```

Reply:
On a second thought I wonder if the message shouldn't be more descriptive/verbose. The following is just an example feel free to change or ignore it. :)
```
Create Knative resources in local directory or file instead of a remote cluster. Default mode is current directory, in addition path to custom direcoty or file extensions .yaml, .yml, .json are accepted. 
```
@rhuss I recall you've mentioned the feature is experimental. Therefore we should add `(experimental)` suffix at the end of description, right?

Reply:
Yes, we should mark it as experimental and we should definitely include it for 0.20

Reply:
To not lose to much time, lets merge this and create a followup PR to add the indication that this feature is experimental in a followup PR.